### PR TITLE
Update Cube.js-Introduction.md

### PR DESCRIPTION
**Pull request by keydunov**

### CODE REVIEW

**Comment by keydunov** [comment]
@igorlukanin @cowperthwait can i get your review?

File: docs/content/Cube.js-Introduction.md [hunk 1]
```diff
@@ -8,11 +8,10 @@ redirect_from:
 
 **Cube is the headless business intelligence platform.** It helps data engineers and application developers access data from modern data stores, organize it into consistent definitions, and deliver it to every application.
 
-Cube.js was designed to work with data warehouses and query engines like Google BigQuery and AWS Athena. A multi-stage querying approach makes it suitable for handling trillions of data points. Most modern RDBMS work with Cube.js as well and can be further tuned for performance.
+Cube was designed to work with all SQL-enabled data sources, including cloud data warehouses like Snowflake or Google BigQuery, query engines like Presto or Amazon Athena, and application databases like Postgres. Cube has a built-in relational caching engine to provide sub-second latency and high concurrency for API requests.
```
Comment (igorlukanin) [inline docs/content/Cube.js-Introduction.md:11]:
```suggestion
Cube was designed to work with all SQL-enabled data sources, including cloud data warehouses like Snowflake or Google BigQuery, query engines like Presto or Amazon Athena, and application databases like Postgres. Cube has a built-in caching engine to provide sub-second latency and high concurrency for API requests.
```

File: docs/content/Cube.js-Introduction.md [hunk 2]
```diff
@@ -8,11 +8,10 @@ redirect_from:
...
 
-With Cube.js, you can create a semantic API layer on top of your data,
-manage access control, cache, and aggregate data. Since Cube.js is visualization agnostic,
-you can use any frontend library to build your own custom UI.
+With Cube, you can create a data model, manage access control and caching, and expose your data to every application via REST, GraphQL and SQL interfaces.
```
Comment (igorlukanin) [inline docs/content/Cube.js-Introduction.md:13]:
```suggestion
With Cube, you can build a data model, manage access control and caching, and expose your data to every application via REST, GraphQL, and SQL APIs. 
```

File: docs/content/Cube.js-Introduction.md [hunk 3]
```diff
@@ -36,15 +35,10 @@ Cube has the necessary infrastructure and features to implement efficient data m
 
 ## Architecture
 
-**Cube.js acts as an analytics backend**, translating business logic (metrics
-and dimensions) into SQL, and managing caching, queuing and database connection.
+**Cube acts as a data access layer**, translating metrics into SQL, managing caching, queuing and database connection.
```
Comment (igorlukanin) [inline docs/content/Cube.js-Introduction.md:38]:
```suggestion
**Cube acts as a data access layer**, translating API requests into SQL, managing caching, queuing, and database connection.
```

File: docs/content/Cube.js-Introduction.md [hunk 4]
```diff
@@ -36,15 +35,10 @@ Cube has the necessary infrastructure and features to implement efficient data m
...
 
-The Cube.js JavaScript client sends queries conforming to the [Query
-Format][ref-query-format] to the REST API. The server uses a Schema to generate
-an SQL query, which is executed by your chosen database. The server handles all
-database connections, as well as pre-aggregations and caching layers. The result
-is then sent back to the client. The client itself is visualization-agnostic and
-works well with any chart library.
+The Cube accepts queries via REST, GraphQL or SQL interfaces. Based on the data model and incoming query, Cube generates
```
Comment (igorlukanin) [inline docs/content/Cube.js-Introduction.md:40]:
```suggestion
The Cube accepts queries via REST, GraphQL or SQL interfaces. Based on the data model and an incoming query, Cube generates
```

File: docs/content/Cube.js-Introduction.md [hunk 5]
```diff
@@ -36,15 +35,10 @@ Cube has the necessary infrastructure and features to implement efficient data m
...
+**Cube acts as a data access layer**, translating API requests into SQL, managing caching, queuing, and database connection.
 
-The Cube.js JavaScript client sends queries conforming to the [Query
-Format][ref-query-format] to the REST API. The server uses a Schema to generate
-an SQL query, which is executed by your chosen database. The server handles all
-database connections, as well as pre-aggregations and caching layers. The result
-is then sent back to the client. The client itself is visualization-agnostic and
-works well with any chart library.
+The Cube accepts queries via REST, GraphQL or SQL interfaces. Based on the data model and incoming query, Cube generates
+SQL query and executes it in your database. The server manages query generation, database connections, as well as caching and access control layers. The result is then sent back to the client.
```
Comment (igorlukanin) [inline docs/content/Cube.js-Introduction.md:41]:
```suggestion
a SQL query and executes it in your database. Cube fully manages query orchestration, database connections, as well as caching and access control layers. The result is then sent back to the client.
```

**Pull request merged**